plugins/fluentd_telemetry_plugin/scripts/upgrade.sh

```diff
@@ -0,0 +1,2 @@
+#!bin/bash
+/bin/usr/python merge_configuration_files.py /config/fluentd_telemetry_conf.cfg /opt/ufm/ufm_plugin_tfs/fluentd_telemetry_plugin/conf/fluentd_telemetry_conf.cfg
```

Comment:
after you move `merge_configuration_files.py` to the sdk_tools repository 
assuming that you pushed the code under `ufm_sdk_tools/src/config_parser_utils/merge_configuration_files.py`
```suggestion
python3 /opt/ufm/ufm_plugin_tfs/ufm_sdk_tools/src/config_parser_utils/merge_configuration_files.py /config/fluentd_telemetry_conf.cfg /opt/ufm/ufm_plugin_tfs/fluentd_telemetry_plugin/conf/fluentd_telemetry_conf.cfg

```

Reply:
done